src/main/java/com/spotinst/sdkjava/model/responses/OceanK8sVirtualNodeGroupApiResponse.java

```diff
@@ -0,0 +1,7 @@
+package com.spotinst.sdkjava.model.responses;
+
+import com.spotinst.sdkjava.client.response.BaseServiceItemsResponse;
+import com.spotinst.sdkjava.model.api.ocean.kubernetes.ApiK8sVirtualNodeGroup;
+
+public class OceanK8sVirtualNodeGroupApiResponse extends BaseServiceItemsResponse<ApiK8sVirtualNodeGroup> {
```

Comment:
Overall comment - Can we replace "VirtualNodeGroup" with VNG in class-names/Filenames and variable names to mae it crisp?